### DISCUSSION TITLE
Welcome to the AI category for Universe 2022!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/events#12] [AI]
🤖 How GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. For questions about this space check out the [FAQ](https://github.com/community/events/discussions/1)  and [tune in live](http://watch.githubuniverse.com/). Let us know what sessions you're attending or loving below 🚀 !

**Comment by ibtihelgharsalah** [comment]
can you watch the live session of copilot? I can not access it by any means !

**Reply by queenofcorgis** [reply]
Thanks for reporting, I know this may not help, but it will be posted on-demand on githubuniverse.com for anyone to watch after the event. Presentations will be available in this repo, post-event.

**Reply by samus-aran** [reply]
You can catch up on all the available sessions after the conference. But to keep watching sessions on the AI track you can [view them here](https://watch.githubuniverse.com/live/dayOneTrackFour).

**Reply by mishmanners** [reply]
The video is also playing live right now and you can use the timeline to go back to the start of the day and listen into the sessions again: https://youtu.be/wKZg0DScucw

**Comment by ameyb22** [comment]
I would like to ask about the codespaces, is there gonna be ml-centric spaces where one can train models?

**Reply by queenofcorgis** [reply]
@otter-computer mind chiming in here when you get back online please?

**Reply by otter-computer** [reply]
I certainly can, thanks @queenofcorgis! 

Codespaces with JupyterLab are in public preview right now, and codespaces with dedicated GPUs are currently in private preview. You can request access to GPU codespaces [with this form](https://github.surveymonkey.com/r/Y75GX9T).

For a look at using Codespaces for ML take a look at [this guide on using Codespaces for Machine Learning](https://docs.github.com/en/codespaces/developing-in-codespaces/getting-started-with-github-codespaces-for-machine-learning) in the docs.

**Comment by homerjam** [comment]
Hi! The GPU beta survey url is down - is it possible to join the program another way?

**Comment by xTalhaAhmedCho** [comment]
![chrome_eTjRY3DcEQ](https://github.com/user-attachments/assets/247865c8-f382-48c5-a43a-de6bd50d3803)

`Dear GitHub Support Team,`

> I am writing to request the re-enablement of GitHub Actions for my account, **@TalhaAhmedCho**. I have noticed that GitHub Actions is currently disabled for my account. 

I understand that my GitHub Actions may have been disabled due to `misunderstanding of limits`. If my actions inadvertently violated any GitHub terms or policies, I sincerely apologize. It was not my intention to misuse the platform. I am committed to using GitHub Actions responsibly and in accordance with all guidelines in the future.

> I value GitHub's services greatly, especially GitHub Actions, which is an **_integral part of my workflow_** for personal project. I rely on GitHub Actions for `learning new technologies. `

**_I kindly request you to review my account and re-enable GitHub Actions. I am willing to cooperate fully and take any necessary steps to ensure I comply with GitHub's policies and guidelines in the future._**

`My GitHub username is:`  **@TalhaAhmedCho**

Thank you for your **_time and consideration_**. I look forward to your positive response and the re-enablement of GitHub Actions.

Sincerely,

Talha Ahmed Chowdhury
[Linkedin](https://www.linkedin.com/in/TalhaAhmedCho/) | [Facebook](https://www.facebook.com/@TalhaAhmedCho) | [YouTube](https://www.youtube.com/@TalhaAhmedCho) | [Twitter](https://x.com/_TalhaAhmedCho)
